### ISSUE TITLE
Cannot reproduce the accuracy of the paper

### ISSUE THREAD

**Issue by rayeeli** [junkeun-yi/SAVi-pytorch#4]
Thank you very much for the pytorch version code! I recently used your code to reproduce SAVi, but I could not reproduce it with the accuracy of the paper. Did you succeed in reproducing it?

**Comment by junkeun-yi** [comment]
Hi, I was able to reproduce the accuracies of MOVi-A and MOVi-B with the small networks.
I've been looking to clean up and fix this repository, but I'm a bit busy and don't know when I'll get to doing it.

**Comment by rayeeli** [comment]
Hey junkeun, thank you for your reply! I am trying to reproduce the experiments in the paper in an unconditional setting, but the performance is far from what is in the paper. On the CATER dataset, SAVi (unconditional) should reach 92.8 ARI-FG, but I only get close to 80 ARI-FG. And on MOVi-A, SAVi (uncond. w/o flow) should reach near 60 ARI-FG. I am not quite sure if it is related to my modification on video.py, since only 2 unconditional initialization modules without config file are provided. Do you have any suggestions for this unconditional setup? Thanks in advance!

**Comment by junkeun-yi** [comment]
You should make sure to match the configurations in the paper, it seems to do two iterations of slot attention, which also means they add an mlp between the slot attention iterations.
It also seems to train for twice the number of timesteps, and I don't know if the their hyperparameters match the MOVi configurations. For details you should ask the authors of the paper.
I'm not sure where the unconditional MOVi-A numbers are from, but my guess is it also changes the configurations

**Comment by rayeeli** [comment]
Thank you very much for your reply.
